src/app/fdctl/config/default.toml

```diff
@@ -512,7 +501,7 @@ dynamic_port_range = "8900-9000"
     [tiles.quic]
         # The period in which there are no active requests.
```

Comment:
Another nit be can we reword this,

```# The duration after which an idle connection will be timed out, and the connection will be closed.  An idle connection is one where no data is received from the peer.```

Also can you wrap to 72 columns. There's a VS code extension that can help with this.